maintainers/maintainer-list.nix

```diff
@@ -23308,6 +23308,11 @@
     githubId = 9870613;
     name = "Hubert Mühlhans";
   };
+  tris203 = {
+    email = "admin@snappeh.com";
```

Comment:
This is missing the `name` attribute: https://hydra.nixos.org/jobset/nixos/unstable-small#tabs-errors